google/cloud/bigquery/model.py

```diff
@@ -433,3 +433,15 @@ def __repr__(self):
         return "ModelReference(project_id='{}', dataset_id='{}', model_id='{}')".format(
             self.project, self.dataset_id, self.model_id
         )
+
+
+def _model_arg_to_model_ref(value, default_project=None):
+    """Helper to convert a string or Model to ModelReference.
+
+    This function keeps ModelReference and other kinds of objects unchanged.
+    """
+    if isinstance(value, six.string_types):
+        value = ModelReference.from_string(value, default_project=default_project)
+    if isinstance(value, Model):
+        value = value.reference
+    return value
```

Comment:
I suppose it's better to return like:
```
return ModelReference.from_string(value, default_project=default_project)
```
instead of setting `value` and returning it on a next line. Instancing a variable takes some time (very-very small, but in Python this time is bigger than in other languages).

Reply:
Ok, then What about `value = value.reference` and just trying to maintain consistency as the same use for _table_arg_to_table_ref which declared in table class

Reply:
I mean, I'd write this these way:
```
  if isinstance(value, six.string_types):
        return ModelReference.from_string(value, default_project=default_project)
    if isinstance(value, Model):
        return value.reference
```
The sooner function returns, the faster it works

Reply:
yes agree, but then i have to write three return statement in three line so, and also follow the same done for table,